test/examples/py2pack-opensuse-augmented.spec

```diff
@@ -75,6 +75,6 @@ CHOOSE: %pytest OR %pyunittest -v OR CUSTOM
 %doc AUTHORS ChangeLog README.rst
 %license LICENSE
 %python_alternative %{_bindir}/py2pack
-%{python_sitelib}/*
```

Comment:
Obviously the same as above.

Reply:
Shouldn't the test expect also the new two line form? The same in the other hunk below.

Reply:
Shouldn't the test expect also the new two line form? The same in the other hunk below.